data/config-schema/src/main/resources/net/corda/schema/configuration/flow/1.0/corda.flow.json

```diff
@@ -53,6 +53,25 @@
           "default": 60000
         }
       }
+    },
+    "db": {
```

Comment:
@JamesHR3 @rick-r3 @driessamyn This is a wider question. 

But are we going to stick to `persistence` or `db` for naming conventions? `persistence` is what we use in the API. `db` is used internally, e.g. `DBWorker`. Having some consistency around these things will probably make things easier, for example, when users are uploading configuration they shouldn't have to think about whether it comes under `db` or `persistence`, it should be obvious because we've chosen to stick to a single wording.

Reply:
After standup discussion i'll rename "db" config field in the flow config to "persistence"